writing/milestone-0/OVERVIEW.md

```diff
@@ -0,0 +1,80 @@
+
+In practice this means a single initial machine with cables that can be plugged into one or more "expansion boxes". People can invest in expanding these systems over many years. With each expansion, the system can assist in more activities and bring more smarts to each of them.
+
+We have plans for single-, double-, and quad-accelerator expansion boxes that can be plugged into a variety of machines. The plans will be available in early 2024. The double reuses all the parts of the single, and the quad reuses all the parts of the single and double.
+Who It's For
+Initially, this project will serve hackers who want to build and collaborate with one another on local and intelligent systems. 
+
+### Who is it for? What are some sample situations?
+
+A1. Hackers (working on an open source software/hardware project)
+
+A2. Students (in grad school doing research, with international and multilingual collaborators)
+
+A3. Small and mid-sized businesses (a regional distributor)
+
+
+### What It Is
+
+#### What you get in the mail:
+- Substrate System 0 is an easy-to-assemble AI computer kit for home and work.
+
+#### Once built, it runs:
+- SubstrateOS, an operating system that boots from a USB drive and schedules AI models on internal and external consumer GPUs.
+
+#### Once booted, it can:
+- M0.1. Join in on real-world conversations via your other devices. [A1] [A2] [A3]
+    A1. Planning meeting
+    A2. Lecture
+    A3. Sales meeting, customer call
+
+- M0.2. Take meeting and lecture notes so you can pay full attention to the speakers. [Liv] [A1] [A2] [A3]
+    A1. Meeting minutes, next steps
+    A2. Lecture notes
+    A3. ?
+
+- M0.3. Identify and link to more info on people, events, and cited documents. [Liv] [A1] [A2]
+    A1. ?
+    A2. Important dates for the course; references to other talks, textbooks, papers, textbooks
+
+- M0.4. Keep an at-hand, ever-present record of conversation, tasks, and specific contributions. This captures good ideas as they happen, foster new connections between them, and helps maintain understanding between collaborators. [Matt] [Liv] [A1] [A2] [A3]
+    A1. Feature suggestions, observations, 
+    A2. ?
+    A3. ?
+
+- M0.5. Eliminate language barriers by translating between any number of supported languages in real-time, across speech and text. [A2] [A3]
+    A2. ?
+    A3. ?
+
+- M0.6. Respond only when actively addressed directly or indirectly, so using its name in a sentence does not result in false positives, but people also don't need to keep using its name explicitly to continue a conversation.
+
+- M0.7. Self-assess its reasoning capabilities so it knows what it can do and how well it performs relative to expectations.
+
+- M0.8. Cast wirelessly to nearby displays. [?Jeff] [A1] [A2] [A3]
+    A1. ?
+    A2. ?
+    A3. ?
+
+- M0.9. Help draft comparative analyses of ideas shared in both documents and live interactions. [A1] [A2]
+    A1. ?
+    A2. ?
+
```

Comment:
A3. If the business in question is a research enterprise, this is going to dramatically accelerate distributed search of blind spots in the space of possible hypotheses/experiment designs. I spoke to James Evans about doing this (https://complexity.simplecast.com/episodes/55) — in his recent work he has been treating "What questions will scientists be asking in a year?" as a token prediction challenge and finding truly novel opportunities by superimposing ideas from different fields. Deploying the machine as a kind of naïve outsider, because many if not most scientific breakthroughs come from people with little expertise in the field where they made their breakthrough...contrasting ideas from, for instance, a ecology paper and an economics paper can reveal opportunities to apply models in new domains, or reveal where the barrier of technical language has obscured a deeper unifying insight independently discovered in multiple areas.  You can also "go hipster" and reverse the polarity of the system's suggestions to identify blind spots common to both human and machine and make a bet on finding something totally unexpected.

Reply:
Given the above, imagine you could task a computer to bring a concrete set of questions to each encounter (written or verbal). What would those questions be?

An example question might be:

- Which parts of this work might still be important in a year? In 5 years? 50 years?

Always answering the same question(s) for every task might get tedious. Feel free to give a context-specific question or a sequence of interrelated questions.